backend/routes/searchDpsUserRouter.test.js

```diff
@@ -59,4 +64,32 @@ describe('Search DPS user router', () => {
       expect(results).toEqual([{ username: 'joey', status: 'active', email: 'joey@bloggs' }])
     })
   })
+
+  describe('caseloads', () => {
+    it('should do nothing if no admin role', async () => {
+      const context = { user: 'bob' }
+      const results = await caseloads(context)
+      expect(results).toEqual([])
+    })
+    it('should pass the context through to the apis', async () => {
+      apis.prisonApi.getCaseloads.mockResolvedValue([])
+      const context = { user: { maintainAccessAdmin: true } }
+      await caseloads(context)
+      expect(apis.prisonApi.getCaseloads.mock.calls[0][0]).toEqual(context)
```

Comment:
Sorry if you already know but you can also do `expect(apis.prisonApi.getCaseloads).toHaveBeenCalledWith(context)`